app/models/manageiq/providers/cisco_intersight/physical_infra_manager/operations/lifecycle.rb

```diff
@@ -0,0 +1,56 @@
+module ManageIQ::Providers::CiscoIntersight
+  module PhysicalInfraManager::Operations::Lifecycle
+    def recommission_server(server, options)
+      _log.info("Requesting server recommission #{server.ems_ref}.")
+
+      with_provider_connection do |_client|
```

Comment:
Not for this PR but I think we'd be better off building a new `IntersightClient::Configuration` and passing that to the `IntersightClient::ApiClient` and yielding that to the block rather than setting the api_key to the `@@default`.  Just reducing any possibility of races when running with multiple intersight providers in the same process (aka GenericWorker)